tuxemon/log.py

```diff
@@ -35,6 +36,14 @@ def configure() -> None:
         warnings.filterwarnings("default")
 
         for logger_name in config.loggers:
+            # Get the current git hash
```

Comment:
If somebody somehow plays the game without git installed will this crash?

Best to wrap it in a try/catch.

Reply:
it makes sense, what do you want in the exception? I put a "No Git Hash", let me know.

Reply:
Yes really great catch.
I think we can add something like "Do you have Git installed?".

Reply:
Wait wait wait, just going back to this piece of code: the computation of the githash should be **inside** of the try/except block, because that's this computation that might fail (not the print operation).